presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -389,6 +396,20 @@ protected Scope visitDelete(Delete node, Optional<Scope> scope)
         }
 
         @Override
+        protected Scope visitCreateSchema(CreateSchema node, Optional<Scope> scope)
```

Comment:
you moved and changed these methods in the same commit, can you please separate such changes to separate commits. It is much easier to follow that way.